clients/cloud/rust/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
This should be just target, right? /target is outside the git repo

Reply:
this `.gitignore` was generated by the cargo cli